example/centos/centos-9-x86_64-vmdk.yaml

```diff
@@ -0,0 +1,121 @@
+otk.version: "1"
+
+otk.define:
+  filesystem:
+    modifications:
+  kernel:
+    cmdline: ro net.ifnames=0
+  packages:
+    build:
+      otk.external.osbuild-gen-depsolve-dnf4:
+        architecture: x86_64
+        module_platform_id: c9s
+        releasever: "9"
+        repositories:
+          otk.include: "common/repositories/x86_64.yaml"
+        packages:
+          include:
+            - coreutils
+            - dosfstools
+            - glibc
+            - grub2-pc
+            - platform-python
+            - policycoreutils
+            - python3
+            - rpm
+            - selinux-policy-targeted
+            - systemd
+            - qemu-img
+            - xfsprogs
+            - xz
+          exclude: []
+    os:
+      otk.external.osbuild-gen-depsolve-dnf4:
+        architecture: x86_64
+        module_platform_id: c9s
+        releasever: "9"
+        repositories:
+          otk.include: "common/repositories/x86_64.yaml"
+        packages:
+          include:
+            - "@core"
+            - chrony
+            - cloud-init
+            - dosfstools
+            - dracut-config-generic
+            - efibootmgr
+            - firewalld
+            - grub2-efi-x64
+            - grub2-pc
+            - kernel
+            - langpacks-en
+            - open-vm-tools
+            - selinux-policy-targeted
+            - shim-x64
+            - tuned
+            - xfsprogs
+          exclude:
+            - dracut-config-rescue
+            - rng-tools
+
+otk.include: "common/partition-table/x86_64/vhd_vmdk.yaml"
+
+otk.target.osbuild:
+  pipelines:
+    - otk.include: "pipeline/build/generic.yaml"
+    - name: os
+      build: name:build
+      stages:
+        - type: org.osbuild.kernel-cmdline
+          options:
+            root_fs_uuid: 9851898e-0b30-437d-8fad-51ec16c3697f
```

Comment:
Same comment as above about using a define here

Reply:
Yes, but when I set the UUIDs to the ones from the partition table generation the tests start failing. I've done so in a separate commit so we can look at the failure.